webhooks-extension/docs/InstallReleaseBuild.md

```diff
@@ -0,0 +1,93 @@
+# Installing Official Releases
+<br/>
+
+Run the [`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply) command to perform installation.  By default the installation will be into a namespace called "tekton-pipelines".  
+
+If the tekton dashboard has been installed into a namespace other than "tekton-pipelines", then you will need to ensure you install the webhooks-extension into the same namespace.  Instructions for installing into an alternative namespace are provided below the standard install instructions.
+<br/>
+<br/>
+
+  * **To install latest:**
```

Comment:
latest images*? Consistent with whatever you go with for `webhooks-extension/docs/InstallNightlyBuild.md`

Reply:
gone with latest release image